data/credits/en.credits

```diff
@@ -1,6 +1,46 @@
 # Lines longer than 78 characters will be wrapped automatically
 # The following line is for reference
 ##############################################################################
+(<color_cyan>0.H</color>):</color>
+<color_white>Original author:</color>    <color_white>Project Manager:</color>
+<color_yellow>Whales</color> (<color_dark_gray>retired</color>)    <color_yellow>KevinGranade</color>
+
+<color_white>Current Main Developers:</color>
+<color_yellow>I-am-Erk</color> - Ultica, lore writing, ideas guy.
+<color_yellow>BrettDong</color> - For translations and analyzing weird bugs.
```

Comment:
I haven't had time to really contribute much other than occasional bug-fixes during this time period, I think it'd better to get my name removed from the credits.

```suggestion
```